.aicoe-ci.yaml

```diff
@@ -1,14 +1,15 @@
 # Setup and configuring aicoe-ci with configuration file `.aicoe-ci.yaml`
 # Example `.aicoe-ci.yaml` with a full list of config options is available here: https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml
 check:
-  # Uncomment following line to build a public image of this repo
-  # - thoth-build
-# Uncomment following lines to build a public image of this repo
-# build:
-#   build-stratergy: Source
-#   build-source-script: "image:///opt/app-root/builder"
-#   base-image: quay.io/thoth-station/s2i-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+  - thoth-precommit
```

Comment:
IIUC precommit tests are already being run through prow [here](https://github.com/aicoe-aiops/operate-first-jupyterhub-analysis/blob/54c2b71fe2cd9a24dd96431c5bfaeae441753c43/.prow.yaml#L2), so I think we can remove this line.